packages/evil-winrm/evil-winrm.install

```diff
@@ -5,6 +5,8 @@ post_install() {
   bundle config build.nokogiri --use-system-libraries
   bundle config set --local path 'vendor/bundle'
   bundle install
+  sed -zi "s/\[provider_sect\]\ndefault = default_sect/\[provider_sect\]\ndefault = default_sect\nlegacy = legacy_sect/" /etc/ssl/openssl.cnf
```

Comment:
As said here https://github.com/BlackArch/blackarch/issues/3593#issuecomment-1326805046, we don't want to modify openssl globally / system-wide, we should find a way to create a custom config for evil-winrm and load this one.